__DEFINES/global.dm

```diff
@@ -353,3 +353,13 @@ var/list/extraMiniMaps = list()
 var/list/holomap_markers = list()
 
 var/holomaps_initialized = 0
+
+//BRUISEPACKS AND SURGERY
+
+#define CAN_MEDICATE_THROUGH_ARMOR FALSE // Ditto
+#define MEDICAL_AID_ARMOR_LIMIT list(melee = 10, bullet = 10, laser = 10,energy = 10, bomb = 10, bio = 10, rad = 10) // Anything tougher than most or all specialized jumpsuits like virologists or security.
+#define PERCENT_SUITABLE_MEDICAL_WORKSPACE 100 // Anything less than a surgery operation table or similar is no good and warns the player
+#define CAN_MEDICATE_THROUGH_PLASMAMEN_SUITS TRUE // Magic I don't know.
```

Comment:
I had the idea to make it a var at amour level, like "var/can_medicate_through` and set it to `true` for plasmamen, but do as you wish.

Also Sonix's argument makes a bit of sense, I hadn't seen things that way.

Reply:
That's a much more graceful way of doing that, yeah.
I'll put that in.